cache-config/testing/docker/trafficserver/Dockerfile

```diff
@@ -19,31 +19,34 @@
 # Dockerfile to build Traffic Server RPM
 ###############################################################
 
-ARG RHEL_VERSION=8
-FROM centos:${RHEL_VERSION}
-ARG RHEL_VERSION=8
+ARG OS_VERSION=8
+ARG OS_DISTRO=rockylinux/rockylinux
```

Comment:
Rocky Linux now has an official image, `rockylinux/rockylinux` can be just `rockylinux`.